### PR TITLE
Eradicate nanoserver and wsc 1803 because they are EOL

### LINKED ISSUE #384: WSC 1803 is EOL, remove the image
cc @RDIL

### PR DESCRIPTION
**Pull request by RDIL**
fixes #384 

Signed-off-by: Reece Dunham <me@rdil.rocks>

### CODE REVIEW

**Comment by RDIL** [comment]
@PoshChan please retry CI

**Comment by PoshChan** [comment]
@RDIL, successfully started retry of `docker-PR`

File: createAllManifests.ps1 [hunk 1]
```diff
@@ -17,27 +17,24 @@ $createScriptPath = Join-Path -Path $PSScriptRoot -ChildPath 'createManifest.ps1
 
 $latestStableUbuntu   = "ubuntu-bionic"
 $latestStableWscLtsc  = "windowsservercore-latest"
-$latestStableWsc1803  = "windowsservercore-1803"
 $latestStableWsc1809  = "windowsservercore-1809"
 $latestStableWsc1903  = "windowsservercore-1903"
-$latestStableNano1803 = "nanoserver-1803"
 $latestStableNano1809 = "nanoserver-1809"
 $latestStableNano1903 = "nanoserver-1903"
```
Comment (TravisEz13) [inline createAllManifests.ps1]: we should add 1909 to the list... it's unrelated... could be a separate PR.

**Comment by TravisEz13** [comment]
sorry this needs to wait on 7.0's release...

**Comment by RDIL** [comment]
thats ok

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 Hour.

**Comment by TravisEz13** [comment]
@RDIL  thank you for all your contributions

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 Hour. ago

**Comment by TravisEz13** [comment]
released

**Comment by RDIL** [comment]
@msftbot[bot] is slacking haha

**Comment by TravisEz13** [comment]
I'd have to create releases...

**Comment by TravisEz13** [comment]
I'm slacking...

**Comment by RDIL** [comment]
No you're not, I'm just saying that ususally msftbot comments those types of 'this has been release' messages haha
